test/speed-index.js

```diff
@@ -136,6 +136,15 @@ test('speed indexes calculated for realistic trace', async t => {
 	t.is(Math.floor(indexes.perceptualSpeedIndex), 578);
 });
 
+test('speed indexes calculated for realistic trace with --fast', async t => {
```

Comment:
let's also assert that we didn't call getPerceptualProgress, etc on every frame, since that's the essence of this.